client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,64 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const PhotoBlogBanner = () => {
+	const onClick = () => {
+		alert( 'Photo Blog banner button clicked!' );
+	};
```

Comment:
If we're adding an action, this shouldn't be a functional component, but rather `extend PureComponent`. However, you might just want to drop the dummy actions for now since we don't want to merge this PR with them anyway (and change to a `PureComponent` later, if/when needed).